src/js/client.js

```diff
@@ -1,9 +1,11 @@
 import React from "react"
 import ReactDOM from "react-dom"
 import { Provider } from "react-redux"
-import Bootstrap from 'bootstrap/dist/css/bootstrap.css';
 import Layout from "./components/Layout";
 import store from "./store";
+import 'bootstrap/dist/css/bootstrap.css';
+
+require("../styles/main-styles.scss");
```

Comment:
Please keep one convension, better would be useing `import "../styles/main-styles.scss"`, also please use in import `"` or `'`